t/opcode-coverage.neon

```diff
@@ -0,0 +1,594 @@
+FUNCTION indexan(): String
+    RETURN rec().field
+END FUNCTION
+
+#    INDEXDR,    // index dictionary for read
+FUNCTION indexdr(): Number
+    RETURN dictionary["hello"]
+END FUNCTION
+
+#    INDEXDW,    // index dictionary for write
+FUNCTION indexdw()
+    dictionary["Neon"] := 20.1797
+END FUNCTION
+
+#    INDEXDV,    // index dictionary value
+FUNCTION indexdv()
+    FUNCTION d(): Dictionary<Number>
+        RETURN dictionary
+    END FUNCTION
+    #print("\(d()[\"hello\"])")
+    print(d()["hello"].toString())
+END FUNCTION
+
+#    INA,        // in array
+FUNCTION ina()
+    IF 5 IN array THEN
+    END IF
+END FUNCTION
+
+#    IND,        // in dictionary
+FUNCTION ind()
+    IF "two" IN dictionary THEN
+    END IF
+END FUNCTION
+
+#    CALLP,      // call predefined
+FUNCTION callp()
+    print("Hello, World!")
+END FUNCTION
+
+#    CALLF,      // call function
+FUNCTION callf()
+    callp()
+END FUNCTION
+
+#    CALLMF,     // call module function
+FUNCTION callmf(): Number
+    RETURN module.function(1)
+END FUNCTION
+
+#    CALLI,      // call indirect
+FUNCTION calli()
+    FUNCTION life()
+    END FUNCTION
+    VAR pLife: FUNCTION() := life
+    pLife()
+END FUNCTION
+
+#    JUMP,       // unconditional jump
+FUNCTION jump()
+    IF 1 = 2 THEN
+    END IF
+END FUNCTION
+
+#    JF,         // jump if false
+FUNCTION jf()
+    FOR i := 1 TO 2 DO
+    END FOR
+END FUNCTION
+
+#    JT,         // jump if true
+FUNCTION jt():Boolean
+    RETURN TRUE OR FALSE
+END FUNCTION
+
+#    JFCHAIN,    // jump and drop next if false
+FUNCTION jfchain(): Boolean
+    RETURN 1 < 2 < 3
+END FUNCTION
+
+#    DUP,        // duplicate
+FUNCTION dup():Boolean
+    VAR x: Boolean := TRUE OR FALSE
+    RETURN x
+END FUNCTION
+
+#    DUPX1,      // duplicate under second value
+FUNCTION dupx1()
+    VAR x: POINTER TO class
+    x := NEW class(Field WITH 42)
+END FUNCTION
+
+#    DROP,       // drop
+FUNCTION drop():Boolean
+    VAR x: Boolean := TRUE OR FALSE
+    RETURN x
+END FUNCTION
+
+#    RET,        // return
+FUNCTION ret()
+END FUNCTION
+
+#    CONSA,      // construct array
+FUNCTION consa(): Array<Number>
+    RETURN [42]
+END FUNCTION
+
+#    CONSD,      // construct dictionary
+FUNCTION consd(): Dictionary<Number>
+    RETURN {"FortyTwo": 42}
+END FUNCTION
+
+#    EXCEPT,     // throw exception
+FUNCTION except()
+    RAISE TestException
+END FUNCTION
+
+#    ALLOC,      // allocate record
+FUNCTION alloc()
+    VAR x: POINTER TO class
+    x := NEW class(Field WITH 42)
+END FUNCTION
+
+#    PUSHNIL,    // push nil pointer
+FUNCTION pushnil(): POINTER TO class
+    VAR x: POINTER TO class := NIL
+    RETURN x
+END FUNCTION
+
+#    RESETC,     // reset cell
+FUNCTION resetc()
+    VAR x: Number
+    x := 1
+END FUNCTION
+
+#    PUSHPEG,    // push pointer to external global
+FUNCTION pushpeg()
+    # This requires the REPL to implement.
+END FUNCTION
+
+#    JUMPTBL,    // jump table
+FUNCTION jumptbl(): Number
+    CASE(1)
```

Comment:
Unnecessary parentheses on this and the following line.

Reply:
Fixed.